bench/bench.go

```diff
@@ -25,7 +26,7 @@ type Benchmark interface {
 	// how the agentNum affects it. Every benchmark should have a 'Name' field
 	// set by init, which appears when the benchmark is marshalled to json as
 	// "name".
-	Init(hosts []string, agentNum int) error
+	Init(hosts []string, agentNum int, clientOptions *pcli.ClientOptions) error
```

Comment:
I don't love adding an argument to this which depends on the client - in theory the benchmark doesn't need to use the client lib at all. I do see the need for more flexible options on Benchmarks though. Perhaps we can replace `hosts []string` with something like `type Pilosa struct { Hosts []string, TLSskipVerify bool, ... }`

Reply:
Added `HostSetup` struct.